Traceback (most recent call last):
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/calvin/miniconda3/envs/meillionen/lib/python3.8/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import itertools
import os.path
import netCDF4
from meillionen.client import ClientFunctionModel

from meillionen.experiment import Experiment, PathSettings, Trial
from meillionen.handlers import NetCDFHandler, PandasHandler
from meillionen.resource import \
    FileResource, FeatherResource, NetCDFResource, ParquetResource
from prefect import task, Flow

os.environ['SIMPLECROP'] = 'simplecrop'
os.environ['RUST_BACKTRACE'] = '1'
BASE_DIR = '../../examples/crop-pipeline'


INPUT_DIR = os.path.join(BASE_DIR, 'workflows/inputs')
OUTPUT_DIR = os.path.join(BASE_DIR, 'workflows/outputs')

experiment = Experiment(
    sinks=PathSettings(base_path=OUTPUT_DIR),
    sources=PathSettings(base_path=INPUT_DIR)
)
trial = experiment.trial("2021-05-26")
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[0;32m/tmp/ipykernel_14840/3668342031.py[0m in [0;36m<module>[0;34m[0m
[1;32m      5[0m [0;34m[0m[0m
[1;32m      6[0m [0;32mfrom[0m [0mmeillionen[0m[0;34m.[0m[0mexperiment[0m [0;32mimport[0m [0mExperiment[0m[0;34m,[0m [0mPathSettings[0m[0;34m,[0m [0mTrial[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 7[0;31m [0;32mfrom[0m [0mmeillionen[0m[0;34m.[0m[0mhandlers[0m [0;32mimport[0m [0mNetCDFHandler[0m[0;34m,[0m [0mPandasHandler[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      8[0m [0;32mfrom[0m [0mmeillionen[0m[0;34m.[0m[0mresource[0m [0;32mimport[0m[0;31m [0m[0;31m\[0m[0;34m[0m[0;34m[0m[0m
[1;32m      9[0m     [0mFileResource[0m[0;34m,[0m [0mFeatherResource[0m[0;34m,[0m [0mNetCDFResource[0m[0;34m,[0m [0mParquetResource[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/miniconda3/envs/meillionen/lib/python3.8/site-packages/meillionen/handlers.py[0m in [0;36m<module>[0;34m[0m
[1;32m      8[0m [0;34m[0m[0m
[1;32m      9[0m [0;32mfrom[0m [0mlandlab[0m[0;34m.[0m[0mio[0m [0;32mimport[0m [0mread_esri_ascii[0m[0;34m,[0m [0mwrite_esri_ascii[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 10[0;31m [0;32mfrom[0m [0mmeillionen[0m[0;34m.[0m[0minterface[0m[0;34m.[0m[0mbytesio[0m [0;32mimport[0m [0mResource[0m[0;34m,[0m [0mSchema[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     11[0m [0;32mimport[0m [0mmeillionen[0m[0;34m.[0m[0minterface[0m[0;34m.[0m[0mSchema[0m [0;32mas[0m [0mschema[0m[0;34m[0m[0;34m[0m[0m
[1;32m     12[0m [0;32mfrom[0m [0mmeillionen[0m[0;34m.[0m[0mmeillionen[0m [0;32mimport[0m [0mDataFrameSchema[0m[0;34m,[0m [0mTensorSchema[0m[0;34m,[0m [0mFileResource[0m[0;34m,[0m [0mFeatherResource[0m[0;34m,[0m [0mNetCDFResource[0m[0;34m,[0m [0mParquetResource[0m[0;34m[0m[0;34m[0m[0m

[0;31mModuleNotFoundError[0m: No module named 'meillionen.interface.bytesio'
ModuleNotFoundError: No module named 'meillionen.interface.bytesio'

